src/main/resources/db/changelog/changes/02-create-users-table.yaml

```diff
@@ -0,0 +1,39 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-users-table
+      author: shastkiv
+      changes:
+        - createTable:
+            tableName: users
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: email
+                  type: varchar(50)
```

Comment:
Better use the same length validators with dto. Otherwise, you may get an exception when saving to the DB.